pom.xml

```diff
@@ -0,0 +1,139 @@
+    </organization>
+
+    <inceptionYear>2021</inceptionYear>
+    <url>http://www.eclipse.org/eclipselink</url>
+
+    <licenses>
+        <license>
+            <name>Eclipse Public License - v 2.0</name>
+            <url>http://www.eclipse.org/legal/epl-2.0</url>
+            <distribution>repo</distribution>
+        </license>
+        <license>
+            <name>Eclipse Distribution License - v 1.0</name>
+            <url>http://www.eclipse.org/org/documents/edl-v10.php</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <issueManagement>
+        <system>github</system>
+        <url>https://github.com/eclipse-ee4j/eclipselink-asm/issues</url>
+    </issueManagement>
+
+    <mailingLists>
+        <mailingList>
+            <name>EclipseLink user's mailing list</name>
+            <post>eclipselink-users@eclipse.org</post>
+            <subscribe>https://accounts.eclipse.org/mailing-list/eclipselink-users</subscribe>
+            <unsubscribe>https://accounts.eclipse.org/mailing-list/eclipselink-users</unsubscribe>
+            <archive>https://dev.eclipse.org/mhonarc/lists/eclipselink-users/</archive>
+        </mailingList>
+        <mailingList>
+            <name>EclipseLink developer's mailing list</name>
+            <post>eclipselink-dev@eclipse.org</post>
+            <subscribe>https://accounts.eclipse.org/mailing-list/eclipselink-dev</subscribe>
+            <unsubscribe>https://accounts.eclipse.org/mailing-list/eclipselink-dev</unsubscribe>
+            <archive>https://dev.eclipse.org/mhonarc/lists/eclipselink-dev/</archive>
+        </mailingList>
+    </mailingLists>
+
+    <repositories>
+        <repository>
+            <id>sonatype-nexus-staging</id>
+            <name>Sonatype Nexus Staging</name>
+            <url>${sonatypeOssDistMgmtStagingUrl}</url>
+            <releases>
+                <enabled>true</enabled>
+            </releases>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
+    </repositories>
+
+    <properties>
+        <!-- TOOL Properties -->
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <!-- PROJECT Properties -->
+        <build.type>SNAPSHOT</build.type>
+        <maven.build.timestamp.format>yyyyMMddHHmm</maven.build.timestamp.format>
+        <build.qualifier>v${maven.build.timestamp}</build.qualifier>
+        <bundle.version>${project.version}.${build.qualifier}</bundle.version>
+        <project.build.testReports.subdirectory>test-reports</project.build.testReports.subdirectory>
+        <project.build.testReports.summaryFile>test-summary</project.build.testReports.summaryFile>
+
+        <!--Common properties-->
+        <common.licenceFiles>LICENSE.md, NOTICE.md</common.licenceFiles>
+
+        <!--Eclipse Dependencies version-->
+        <eclipselink.asm.version>9.1.0</eclipselink.asm.version>
```

Comment:
rename this property to `asm.version`

Reply:
the `org.eclipse.persistence.asm` module pom declares `asm.version` already:
```xml
        <asm.version>9.1</asm.version>
```
From my understanding, `asm.version` refers to the ASM project version, but `eclipselink.asm.version` refers to this projects version.

Reply:
there is no relationship between these two pom files

Reply:
Fine, Ill just get rid of the parent pom that you don't want